src/v/cluster/types.h

```diff
@@ -2398,9 +2400,6 @@ struct partition_assignment_cmp {
 using assignments_set
   = absl::btree_set<partition_assignment, partition_assignment_cmp>;
 
-using replication_factor_type
-  = named_type<uint16_t, struct replication_factor_type_tag>;
-
```

Comment:
it seems like the removal of this type will cause the commit to longer compile?

Reply:
oh, this is the double declaration from an earlier commit (there's a previous one way further up in the file) -- so once that earlier commit is amended this goes away.